src/palace/manager/feed/admin.py

```diff
@@ -22,13 +26,16 @@ def suppressed(
 
         q = (
             _db.query(LicensePool)
```

Comment:
Minor: Does this query actually need to join against `LicensePool` anymore? It seems like we're only using the pool to get the `Work`.

Reply:
We need the license pool for the filter conditions. But I could probably return the works directly and at least avoid the additional assignment to get them.

Reply:
Updated.